src/Blacklist/ArrayProvider.php

```diff
@@ -11,34 +11,17 @@
 
 namespace Rollerworks\Bundle\PasswordStrengthBundle\Blacklist;
 
+use Rollerworks\Component\PasswordStrength\Blacklist\ArrayProvider as BaseArrayProvider;
+
 /**
  * Array Blacklist Provider.
  *
  * Provides the blacklist from an array.
  *
  * @author Sebastiaan Stok <s.stok@rollerscapes.net>
+ *
+ * @deprecated since 1.6, to be removed in 2.0. Use {@link BaseArrayProvider} instead.
```

Comment:
you should also add `@trigger_error(..., E_USER_DEPRECATED);`, so that the deprecation is reported to people using the class.